### PR TITLE
Add Melos

### PR DESCRIPTION
**Pull request by jamesblasco**
Adds melos, format files and updates to last version published

### CODE REVIEW

**Comment by remonh87** [comment]
Can you. squash the commits a bit :)?

File: melos_stripe.iml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (remonh87) [inline melos_stripe.iml:1]: Is this file needed?

File: stripe/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+ - **FEAT**: first melos version.
```
Comment (remonh87) [inline stripe/CHANGELOG.md:3]: I do not think this is a user facing feature maybe call it something like: "bug fixes"

**Review by remonh87 — APPROVED** [review]
for the rest looks good to me check my comments and then ship it :)

File: melos_stripe.iml (hunk 1 above)
Comment (jonasbark) [reply melos_stripe.iml:1]: yeah, should be removed imo
